sdk/storage/examples/blob_00.rs

```diff
@@ -21,20 +21,33 @@ async fn main() -> Result<(), Box<dyn Error>> {
         .nth(2)
         .expect("please specify blob name as command line parameter");
 
-    let client = client::with_access_key(&account, &master_key);
+    let http_client: Arc<Box<dyn HttpClient>> = Arc::new(Box::new(reqwest::Client::new()));
+
+    let storage_account_client =
+        StorageAccountClient::new_access_key(http_client.clone(), &account, &master_key);
+    let storage_client = storage_account_client.as_storage_client();
+    let blob_client = storage_client
+        .as_container_client(&container)
+        .as_blob_client(&blob);
 
     trace!("Requesting blob");
 
-    let response = client
-        .get_blob()
-        .with_container_name(&container)
-        .with_blob_name(&blob)
-        .finalize()
+    let response = blob_client
+        .get()
+        .range(Range::new(0, 128000))
+        .execute()
         .await?;
 
-    let s_content = String::from_utf8(response.data)?;
-    println!("blob == {:?}", blob);
-    println!("s_content == {}", s_content);
+    println!("{:#?}", response);
+
+    let mut stream = Box::pin(blob_client.get().stream(128));
+    while let Some(value) = stream.next().await {
+        println!("received {:?} bytes", value?.len());
+    }
+
+    //let s_content = String::from_utf8(response.data)?;
+    ////println!("blob == {:?}", blob);
+    ////println!("s_content == {}", s_content);
```

Comment:
Should these be uncommented?

Reply:
You are right, leaving unfinished code in examples is *really* bad practice 🤦‍♂️ .. Fixed!